x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/operator/exchange/ExchangeService.java

```diff
@@ -50,8 +50,10 @@ public final class ExchangeService extends AbstractLifecycleComponent {
     // TODO: Make this a child action of the data node transport to ensure that exchanges
     // are accessed only by the user initialized the session.
     public static final String EXCHANGE_ACTION_NAME = "internal:data/read/esql/exchange";
+    public static final String EXCHANGE_ACTION_NAME_FOR_CCS = "cluster:internal:data/read/esql/exchange";
```

Comment:
I think we can rename the current action names as legacy and private and register them for BWC purposes only?

Reply:
If we rename the "internal:" actions names it would implications for local actions, and internal is correct for the local cluster. This is a bit of a hack, but it is also how the other internal actions that have to go cross cluster are modeled.